esquery.js

```diff
@@ -308,7 +390,7 @@
          * Parse a selector string and return its AST.
          */
         function parse(selector) {
-            return parser.parse(selector);
+            return transform(parser.parse(selector));
```

Comment:
This is my main concern about the changes. Is the transform necessary? Or is it just preventing this from being a breaking change? Could we just stop supporting the subject indicator?

Reply:
It's just to prevent this from being a breaking change.
The subject indicators is deprecated for a long time now, It maybe a good thing to remove it.
I can make the change, if needed.

Reply:
I think it would be okay to make this a breaking change, I can imagine there being issues with the transform which might be annoying. Perhaps we can just document how a user would need to change their query?

Reply:
It's sounds like a good idea, but I we drop the support for `subject indicator` we probably want to change the parser too and remove the related tests no ?

Reply:
Probably, I guess we could do that as a separate PR though?